### ISSUE TITLE
Memory leaks after resizing window with software rendering enabled.

### ISSUE THREAD

**Issue by tomaszte** [f1nalspace/final_game_tech#5]
Hello, after resizing the window memory usage is increased (when I'm starting resizing and holding there is 100mb/s added), there is no such problem with settings.video.isAutoSize option disabled.

Code where it reproduces for me:

```
#define FPL_IMPLEMENTATION
#include "final_platform_layer.h"

int main(int argc, char **args)
{
	fplSettings settings;
	fplSetDefaultSettings(&settings);
    settings.video.driver = fplVideoDriverType_Software;
    //settings.video.isAutoSize = false;
	if (!fplPlatformInit(fplInitFlags_Video, &settings))
	{
		return -1;
	}

    fplVideoBackBuffer *backBuffer = fplGetVideoBackBuffer();
	backBuffer->useOutputRect = false;

    backBuffer->outputRect.x = 0;
    backBuffer->outputRect.y = 0;
    backBuffer->outputRect.width = backBuffer->width;
    backBuffer->outputRect.height = backBuffer->height;

	while (fplWindowUpdate()) {
	    fplWindowSize windowArea;
	    fplGetWindowArea(&windowArea);
        
        // ... Modify the pixels here (Draw call)
        
        for (uint32_t y = 0; y < backBuffer->height; ++y)
        {
            uint32_t *p = (uint32_t *)((uint8_t *)backBuffer->pixels + y * backBuffer->lineWidth);
            for (uint32_t x = 0; x < backBuffer->width; ++x)
            {
                uint32_t color = x *256 + y * 256 * 256;
                *p++ = color;
            }
        }

	    fplVideoFlip();
	}
}
```

Cheers :)

**Comment by f1nalspace** [comment]
Oh thanks for catching that! I somehow managed to pass MEM_FREE as a parameter to VirtualFree, which is totally wrong. This will result that any memory which gets allocated will never be freed -> Great job final - NOT!

Okay i fixed it in the develop branch and in the next release which will come very soon this is merged into the master branch. Really thanks for finding that one ;)

I will close that issue after the release.

**Comment by f1nalspace** [comment]
Solved in v0.7.7.0
